### ISSUE TITLE
Report Overlap when executing in parallel

### ISSUE THREAD

**Issue by Mfaisalansari** [anshooarora/extentreports-java#707]
@anshooarora 

I am facing an issue when executing test in parallel say i execute 5 test in parallel from 1-5, the report for all the test come under one test and other four test do not log even one step. please help me out.

**Comment by email2vimalraj** [comment]
@Mfaisalansari : You should share some sample for us to help you. Without that it would be very difficult for us to help you further. And moreover, handling parallelism are solely in your hands :)

**Comment by anshooarora** [comment]
Agreed.  How you handle parallel execution depends on your wrapper/usage.  `ExtentTest` has been abstracted out which you can manage per thread.  

I am closing this issue as there was no response from OP.  Please feel free to open a new ticket if you continue to face issues.
